modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -1362,4 +1363,8 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
   getRecoveryProvider(apiToken?: string): RecoveryProvider {
     return forCoin(this.getChain(), apiToken);
   }
+
```

Comment:
Should we push this all the way down to btc, since we only support this on btc so far? We will probably do litecoin support at some point, but wouldn't expect us to do other utxo coins (even though doginals are a thing, they are totally different scripts). (can be follow-up PR too)

Reply:
Ok, i think we should. I can quickly do it in this PR.